docs/developer/.scripts/33_render_status.py

```diff
@@ -170,3 +171,24 @@ def render_e2e_progress():
     formatted_percent = f'{percent:.2f}'
     lines[2] = f'[={formatted_percent}% "{formatted_percent}%"]'
     return lines
+
+
+def render_process_signatures_progress():
+    exclude = {'datadog_checks_base', 'datadog_checks_dev', 'datadog_checks_downloader', 'snowflake'}
```

Comment:
Excluded elements should be a constant list ?

Reply:
It's technically a constant set :p